browser/src/control/Parts.js

```diff
@@ -181,7 +181,7 @@ L.Map.include({
 
 	_addPreviewToQueue: function(part, tileMsg) {
 		for (var tile in this._previewQueue)
-			if (tile[0] === part)
+			if (this._previewQueue[tile][0] === part)
```

Comment:
Ah - interesting; a C++ vs. JS iterator confusion on JS arrays ? - I wonder if there are others of those around (?) =)

Reply:
Yes, it happened to me, I did not see another, but we will know =)